website/source/docs/config/index.html.md

```diff
@@ -739,6 +742,41 @@ LANGUAGE plpgsql;
 
 More info can be found in the [PostgreSQL documentation](http://www.postgresql.org/docs/9.4/static/plpgsql-control-structures.html#PLPGSQL-UPSERT-EXAMPLE):
 
+#### Backend Reference: Cassandra (Community-Supported)
+
+The Cassandra backend has the following options:
+
+  * `hosts` (optional) – Comma-separated list of Cassandra hosts to connect to.
+    Defaults to `"127.0.0.1"`.
+
+  * `keyspace` (optional) – Cassandra keyspace to use. Defaults to `"vault"`.
+
+  * `table` (optional) – Table within the `keyspace` in which to store data.
+    Defaults to `"entries"`.
+
+  * `consistency` (optional) – Consistency level to use when reading/writing data
+    in Cassandra. If set, must be one of `"ANY"`, `"ONE"`, `"TWO"`, `"THREE"`, `"QUORUM"`,
+    `"ALL"`, `"LOCAL_QUORUM"`, `"EACH_QUORUM"`, or `"LOCAL_ONE"`. Defaults to `"LOCAL_QUORUM"`.
+
+You need to ensure the keyspace and table exist in Cassandra:
+
+```cql
+CREATE KEYSPACE "vault" WITH REPLICATION = {
```

Comment:
How about unquoted CQL identifiers? CQL names are lower-case by default but enforcing quoting makes names always case-sensitive. Quoted table/keyspace names could be configured within the table name if quoting is required.